app/src/matchmaking/match/match.cpp

```diff
@@ -323,8 +323,8 @@ bool Match::isUciMove(const std::string& move) noexcept {
 }
 
 void Match::verifyPvLines(const Player& us) {
-    const static auto verifyPv = [](Board board, const std::vector<std::string>& tokens, std::string_view info) {
-        const auto fen = board.getFen();
+    const static auto verifyPv = [this](Board board, const std::vector<std::string>& tokens, std::string_view info) {
```

Comment:
you cant capture this because it is static

Reply:
and even then this won't work because the uci_moves are not correct...

Reply:
yeah, im trying to find a way to get the fen and moves from the command builder